BUILD.md

```diff
@@ -174,6 +174,7 @@ cuML's cmake has the following configurable flags available:
 | BUILD_CUML_MG_TESTS | [ON, OFF]  | ON  |  Enable/disable building cuML algorithm test executable `ml_mg_test`. |
 | BUILD_PRIMS_TESTS | [ON, OFF]  | ON  | Enable/disable building cuML algorithm test executable `prims_test`.  |
 | BUILD_CUML_EXAMPLES | [ON, OFF]  | ON  | Enable/disable building cuML C++ API usage examples.  |
+| BUILD_CUML_BENCH | [ON, OFF] | OFF | Enable/disable building oc cuML C++ benchark.  |
```

Comment:
Why OFF by default? In my quick compilation testing it didn't add much compilation overhead if at all, so we might want to consider making it ON by default

Reply:
I was concerned about it affecting our CI build times. But if you say its not much of an overhead, I'm more than happy to enable this by default. Will do it now.